lib/ttnt/test_selector.rb

```diff
@@ -11,11 +11,12 @@ class TestSelector
 
     # @param repo [Rugged::Reposiotry] repository of the project
     # @param target_sha [String] sha of the target object
+    #   (nil means to target current working tree)
     # @param test_files [#include?] candidate test files
     def initialize(repo, target_sha, test_files)
       @repo = repo
       @metadata = MetaData.new(repo, target_sha)
-      @target_obj = @repo.lookup(target_sha)
+      @target_obj = target_sha ? @repo.lookup(target_sha) : nil
```

Comment:
Cool, thank you! Fixed: https://github.com/Genki-S/ttnt/commit/5331c064d22c9919d0c7816a711fe69840a0c9ad